assets/app/view/player.rb

```diff
@@ -241,5 +245,14 @@ def render_company(company)
         h(:td, td_props, @game.format_currency(company.revenue)),
       ])
     end
+
+    def liquidity
+      value = @player.cash
+      @player.shares_by_corporation.reject { |_, s| s.empty? }.each do |corporation, _|
```

Comment:
liquid = @player
.shares_by_corporation
.sum { |c, _|  @game.round.get_liquid_bundles(@player, corporation).max_by(&:price)&.price || 0 }
@player.cash + liquid

maybe put this in the player object